packages/deployment/app/cumulus_api_distribution.config.yml

```diff
@@ -0,0 +1,10 @@
+default:
+
+  apiDeploy: true
+
+  apis:
+    - name: distribution
+
+  lambdas: !!files [
+    'node_modules/@cumulus/api/config/distribution.yml'
```

Comment:
I realize this isn't from your PR as it was in the api default config originally, but I feel like this configuration file needs a better name, e.g. distribution_lambdas